package.json

```diff
@@ -7,7 +7,7 @@
     "doc": "docs"
   },
   "scripts": {
-    "lint": "eslint -- --fix frontend/src/**/*js frontend/cypress/**.js && stylelint frontend/src/**/*.scss",
+    "lint": "eslint -- --fix frontend/src/**/*js frontend/cypress/**.js && stylelint frontend/src/**/*.scss && pug-lint frontend/src/*.pug frontend/src/tabs/*.pug",
```

Comment:
I think the command can be simplified to `pug-lint frontend/src/**/*.pug` 
instead of `pug-lint frontend/src/*.pug frontend/src/tabs/*.pug` ?

The double asterisk allow us to search recursively in subdirectories.

Reply:
I agree with you but then it doesn't check for `frontend/src/*.pug` cause it is looking for `.pug` files in a subdirectory and not the same directory. 

Steps to check:

1) Create a style error in index.pug
2) Run the command `npm run lint` with this in package.json `pug-lint frontend/src/**/*.pug`

Expected Result: Error
Actual Result: No error

Reply:
Sorry, I made a mistake.

The `**` syntax works as intended when I run `pug-lint` directly via the bash shell.
![image](https://user-images.githubusercontent.com/35621759/74715547-90a40980-5267-11ea-8994-b94ae9164338.png)

But it doesn't work when I run `npm run lint`.

After some reading (see [here](https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784)), it seems the `**` syntax is supported by bash shell, but npm uses the `sh` shell instead (which does not support the `**` expansion).

I also found that `pug-lint` itself does not support this syntax (see [here](https://github.com/pugjs/pug-lint/issues/144)).
In this case, we can just keep the current command.